dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/InDevelopmentPavedRoadFramework.kt

```diff
@@ -31,44 +23,6 @@ abstract class InDevelopmentPavedRoadFramework(
 ) :
     PavedRoadFramework(identifier, label, explanation, frameworkTemplateCsvFile) {
 
-    override fun convertExcelTemplateToToHighLevelComponentRepresentation(
```

Comment:
The corresponding method in the `PavedRoadFramework` class does no longer need to be open. Or should it be for maximum flexibility?

Reply:
Looking on what this function is doing. I feel like we should not delete it. What if we are implementing another framework inheriting from `InDevelopmentPavedRoadFramework` we would not notice the issue, right? Is there a different mechanism in place which notifies us about this.

Reply:
I am not sure. Let's have a call about this.

Reply:
In our call you said that you got this now. So can you close this pls?